go.mod

```diff
@@ -79,6 +80,7 @@ require (
 	github.com/Azure/go-autorest/autorest/date v0.3.0 // indirect
 	github.com/Azure/go-autorest/logger v0.2.1 // indirect
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
+	github.com/DataDog/zstd v1.4.5 // indirect
```

Comment:
this repo contains a lot of C code -- can you double check that we are not becoming reliant on CGO for this functionality? (our build process requires that CGO is disabled, so if there was an issue I would presume that our snapshot build would fail... so I don't think there is an issue)

Reply:
I'm not quite sure how to check all possible permutations for this, but explicitly running `CGO_ENABLED=0 go build -o main ./cmd/syft` works fine

Reply:
A deeper dive resulted in this being included because the `go-rpmutils` has a conditional if CGO enabled, if not enabled, then a pure go library is used: https://github.com/sassoftware/go-rpmutils/blob/e5cf0f808585c10c36b3c133f0a524b93afd34d1/uncompress_nocgo.go#L1-L9